university-task-02/src/pages/Common/Column/Column.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Column.css';
+
+
+const Column = ({ items }) => {
```

Comment:
Such a component name is useless. I see the list of buttons, why it's name is Column?